### DISCUSSION TITLE
希望在播放栏新增一个播放列表按钮，显示当前播放歌曲所在的列表情况。

### DISCUSSION THREAD

**Discussion by glamout** [lyswhut/lx-music-desktop#1743] [Ideas]
场景描述：
选择某个歌单或者榜单开始播放后，如果离开当前界面做了一些操作后，就没法再返回原始榜单或者歌单列表了。
比如，依次操作排行榜-小蜗音乐-热歌榜，开始播放，在播放过程中，如果离开了当前榜单做了别的查询、查看操作，或者点选了小枸、小秋等，查看其他榜单歌曲情况后，再回到小蜗，是不能回到原来在播放的热歌榜，并按榜单列表持续播放的。
建议在最下方播放栏中再增加一个列表按钮，在选择了播放某个歌单或者歌榜后，该列表中会自动加载该榜单歌曲列表，并根据播放情况显示当前播放的歌曲。这样，无论在界面上做任何操作，对不会影响当前在播放的榜单，除非选择了播放其他榜单。
附图为网易云音乐网页播放界面，供参考。
<img width="1004" alt="iShot_2024-01-11_11 01 47" src="https://github.com/lyswhut/lx-music-desktop/assets/39193339/14ae7c6a-30c9-474b-acc4-a2ad51ba553b">

**Comment by buxh2017** [comment]
希望增加能从当前播放歌曲跳转到当前歌曲所在了列表的功能，经常会遇到想收藏正在播放的歌单，但是已经回不到原始歌单页面，没有办法收藏
